client/src/components/Profiles.js

```diff
@@ -128,153 +110,216 @@ const Profiles = () => {
+                    class="px-6 py-3 text-left text-sm font-medium text-gray-500 tracking-wider whitespace-nowrap"
+                  >
+                    Last Name
+                  </th>
+                  <Th>Phone</Th>
+                  <Th>Email</Th>
+                  <Th>Groups</Th>
+                  <Th>Volunteer</Th>
+                  <Th>Status</Th>
+                </tr>
+              </thead>
+              <tbody class="bg-white divide-y divide-gray-200">
+                {filteredProfiles.map(profile => (
+                  <tr onClick={() => history.push(`/profiles/${profile.id}`)}>
+                    <Td>{`${profile.first_name}`}</Td>
+                    <Td>{`${profile.last_name}`}</Td>
+                    <td className="px-5 py-3 whitespace-nowrap">
+                      {profile.phone_number}
+                    </td>
+                    <Td>{profile.email}</Td>
+                    <Td>{getProfilesGroups(profile.id).join(", ")}</Td>
+                    <Td>{profile.type === "volunteer" ? "Yes" : "No"}</Td>
+                    <Td>
+                      <button
+                        style={{ minWidth: "5rem" }}
+                        className={`py-2 px-3 rounded-md font-medium ${getClassNames(
+                          profile.status
+                        )}`}
+                      >
+                        {profile.status}
+                      </button>
+                    </Td>
+                  </tr>
+                ))}
+              </tbody>
+            </table>
+          )}
+        </div>
+        <div className="my-20">
+          <h2 className="mb-5 text-xl font-semibold">This month</h2>
+          <div className="flex space-x-20 justify-between text-center">
+            <MonthlyTotal
+              name="New People"
+              count={totalNewProfiles}
+              colour="border-yellow-200"
+            />
+            <MonthlyTotal
+              name="Active People"
+              count={totalActiveProfiles}
+              colour="border-pink-300"
+            />
+            <MonthlyTotal
+              name="Inactive People"
+              count={totalInactiveProfiles}
+              colour="border-red-400"
+            />
+            <MonthlyTotal
+              name="Volunteers"
+              count={totalVolunteerProfiles}
+              colour="border-blue-200"
+            />
+          </div>
+        </div>
+      </div>
     </>
   );
 };
 
+const MonthlyTotal = ({ name, colour, count }) => (
+  <div className={`flex-1 pb-4 border-b-4 ${colour}`}>
+    <div className="text-2xl font-bold mb-4">{count}</div>
+    <h4 className="text-xl font-semibold text-gray-700">{name}</h4>
+  </div>
+);
+
+const Th = ({ children }) => (
+  <th
+    scope="col"
+    class="px-6 py-3 text-left text-sm font-medium text-gray-500 tracking-wider"
+  >
+    {children}
+  </th>
+);
+
+const Td = ({ children }) => <td className="px-5 py-3">{children}</td>;
+
+const Search = props => (
```

Comment:
maybe consider moving some of these components (at least Search) to new file .. unless you think they will only live within the Profiles component